apps/client/src/features/app-settings/panel/sources-panel/import-map/ImportMapForm.tsx

```diff
@@ -13,21 +13,21 @@ import { convertToImportMap, getPersistedOptions, NamedImportMap, persistImportM
 import style from '../SourcesPanel.module.scss';
 
 interface ImportMapFormProps {
-  hasErrors: boolean;
   isSpreadsheet: boolean;
   onCancel: () => void;
   onSubmitExport: (importMap: ImportMap) => Promise<void>;
   onSubmitImport: (importMap: ImportMap) => Promise<void>;
 }
 
 export default function ImportMapForm(props: ImportMapFormProps) {
-  const { hasErrors, isSpreadsheet, onCancel, onSubmitExport, onSubmitImport } = props;
+  const { isSpreadsheet, onCancel, onSubmitExport, onSubmitImport } = props;
```

Comment:
question: I am not sure we can remove the `hasErrors` property here. it seems that it allows the consumer to give a error state that is global to the flow. I suggest reverting to the code that existed previously